### PR TITLE
remove unused vars

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**Changes**
remove unused vars from default files.

ping @PhilipGough

### CODE REVIEW

File: configure-buildfarm/defaults/main.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 
 project_name: digger
-buildfarm_version: "2.0"
```
Comment (PhilipGough) [inline configure-buildfarm/defaults/main.yml]: is it ok to remove this yeah?

Comment (laurafitzgerald) [reply configure-buildfarm/defaults/main.yml]: this is the only reference to the variable in this installation. For rhmap this value is populated by the vars/buildfarm.yml. See references [here](https://github.com/fheng/rhmap-ansible/search?utf8=%E2%9C%93&q=buildfarm_version&type=)

**Comment by laurafitzgerald** [comment]
@PhilipGough could you take a look at the new changes here. 
As far as I can see from [here](https://github.com/aerogear/digger-installer/blob/a50b60166501855ac20cf9e33e91843f9491e712/provision-osx/tasks/configure_buildfarm_node.yml#L40) `credential_public_key_path` and `credential_private_key_path` are now outdated and it should be `credential_public_key` and `credential_private_key`

**Comment by laurafitzgerald** [comment]
@matskiv could you also take a look here if you get some time. It's not essential to remove them but as they are unused it would be nice to have the repo as clean as possible.

**Pull request merged**